includes/REST_API/Stories_Controller.php

```diff
@@ -93,6 +105,178 @@ public function prepare_item_for_response( $post, $request ) {
 		return apply_filters( "rest_prepare_{$this->post_type}", $response, $post, $request );
 	}
 
+	/**
+	 * Pre-fills story with demo content.
+	 *
+	 * @return string Pre-filled post content if applicable, or the default content otherwise.
+	 */
+	private function get_demo_content() {
+		$file = WEBSTORIES_PLUGIN_DIR_PATH . 'includes/data/stories/demo.json';
+
+		if ( ! is_readable( $file ) ) {
+			return '';
+		}
+
+		$file_content = file_get_contents( $file ); // phpcs:ignore WordPress.WP.AlternativeFunctions.file_get_contents_file_get_contents, WordPressVIPMinimum.Performance.FetchingRemoteData.FileGetContentsUnknown
+
+		if ( ! $file_content ) {
+			return '';
```

Comment:
```suggestion
		$file = WEBSTORIES_PLUGIN_DIR_PATH . 'includes/data/stories/demo.json';

		if ( is_readable( $file ) ) {
			$file_content = file_get_contents( $file ); // phpcs:ignore WordPress.WP.AlternativeFunctions.file_get_contents_file_get_contents, WordPressVIPMinimum.Performance.FetchingRemoteData.FileGetContentsUnknown
			if ( $file_content ) {
			   return $file_content;
			}
		}

		return '';
```

Make the code more testable.